website/sidebars.js

```diff
@@ -26,6 +26,16 @@ module.exports = {
           },
         ],
       },
+      {
+        "Plugins": [
+          "software/plugins/plugins",
+          "software/plugins/store-forward-plugin",
+          "software/plugins/range-test-plugin",
+          "software/plugins/environment-plugin",
+          "software/plugins/ext-notif-plugin",
+          "software/plugins/serial-plugin",
+        ],
+      },
```

Comment:
Please move this section below `Meshtastic-python` or elsewhere as I'm trying to keep like items grouped

Reply:
Done